sdk-scheme-adapter/v2_0_0/backend/openapi.yaml

```diff
@@ -1241,9 +1241,14 @@ components:
     transactionRequestResponse:
       description: A response to a request for a quote.
       properties:
+        homeR2PTransactionId:
```

Comment:
I think this should be on the request, and not the response?

However, there should be a `homeTransactionId` instead.

Reply:
Done

Reply:
As discussed, this should actually be `homeR2PTransactionId`